e2e/e2etests/test_precompiles_distribute.go

```diff
@@ -2,166 +2,203 @@ package e2etests
 
```

Comment:
we can rename the file to be consistent with test name `distribute_and_claim`

Reply:
Done